src/Workspaces/Core/Portable/Utilities/Operations/OperationScopeProgressTracker.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.Utilities
+{
+    /// <summary>
+    /// Wrapper around an IOperationScope to bridge it to an IProgressTracker.
+    /// </summary>
+    internal class OperationScopeProgressTracker : IProgressTracker
```

Comment:
this is to help bridge between our internal type, and the more modern type.  over time we could remove IPRogressTracker from more and more places and use the new IOperationContext/Scope.